### PR TITLE
Statospheric harness fix

### PR DESCRIPTION
**Pull request by PapaGuacamole**
Created separate functions, mmapper_achaea_harness_add_exits (fired from event 'mmapper went outside') and mmapper_achaea_harness_remove_exits (fired from event 'mmapper went inside').

(Not ready for review - Still some tidying to do, just trying out this branch business)

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -3296,6 +3296,47 @@ end</script>
                     </regexCodePropertyList>
                 </Trigger>
             </TriggerGroup>
+            <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
+                <name>City has orb of confinement</name>
+                <script>local cityIDs = {
+	49,		-- Ashtan
```
Comment (vadi2) [inline mudlet-mapper.xml]: Does this work both with crowdmap and the IRE map?

Comment (PapaGuacamole) [reply mudlet-mapper.xml]: I would guess not with the IRE map as I imagine it doesn't use the same area ID. I'll look into something to do with the gmcp info too

Comment (PapaGuacamole) [reply mudlet-mapper.xml]: Can anyone provide me with the IRE map area ID for Targ (enemied!)

Comment (vadi2) [reply mudlet-mapper.xml]: I can't right now, but if you download the IRE map and call the area list function, it would be there

Comment (PapaGuacamole) [reply mudlet-mapper.xml]: I've added in the necessary to get it to work with different area IDs for Targossas dependent on mpp.settings.crowdmap

**Pull request closed without merging**